tests/react/client/components/FAB/index.test.tsx

```
@@ -0,0 +1,46 @@
+    const fabButton = screen.getByRole('button', { name: /\+/ })
+    fireEvent.click(fabButton)
+    const settingsButtons = screen.getAllByRole('button')
+    // The last button in the menu is the settings button
+    const settingsButton = settingsButtons[settingsButtons.length - 1]
+    fireEvent.click(settingsButton)
+    expect(screen.getByTestId('mock-drawers')).toHaveTextContent('settings')
+  })
+})
+
+// Named export for FAB to comply with codebase rules
+export { } // Ensures this is a module
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)